### DISCUSSION TITLE
About the starting value of <index>

### DISCUSSION THREAD

**Discussion by YenShihHung** [EDCD/EDDI#2672] [Troubleshoot Scripts] [answered]
Regarding the starting value of <index>, is it 0 or 1?

<img width="998" alt="截圖 2024-12-24 下午6 17 48" src="https://github.com/user-attachments/assets/a06a10d5-89be-4847-80b6-436418d4ecca" />
<img width="995" alt="截圖 2024-12-24 下午6 18 28" src="https://github.com/user-attachments/assets/aa3619a4-18c7-44ec-84ca-63656c620c19" />

**Comment by Tkael** [comment]
It is zero.

**Reply by YenShihHung** [reply]
In the VoiceAttack file, the project says it starts from 0, but the description in the function starts from 1, which actually starts from 1.

![圖片](https://github.com/user-attachments/assets/f57e1c2c-eeca-4a14-a4ea-6a0118875e74)

**Reply by Tkael** [reply, marked as answer]
Ah I understand. This is an unintended inconsistency that will be corrected in the next release.
